docs/podman-run.1.md

```diff
@@ -224,11 +224,11 @@ The **/etc/resolv.conf** file in the image will be used without changes.
 
 **--dns-option**=*option*
 
-Set custom DNS options
+Set custom DNS options. Invalid if using **--dns-option** with **--network** that is not set to host.
```

Comment:
I think this is 
Invalid if **--network** is set to host.
Is it invalid also if --network is set to a different container?
Is it invalid if --network is set to None?